app/src/openxr/cpp/OpenXRLayers.cpp

```diff
@@ -240,4 +240,34 @@ OpenXRLayerEquirect::Update(XrSpace aSpace, const XrPosef &aPose, XrSwapchain aC
   }
 }
 
+// OpenXRLayerPassthrough;
+
+OpenXRLayerPassthroughPtr
+OpenXRLayerPassthrough::Create(const VRLayerPassthroughPtr& aLayer) {
+  // @FIXME: Consider making this method an actual constructor. Same for similar methods in
+  //         the other layers above.
+  auto result = std::make_shared<crow::OpenXRLayerPassthrough>();
+  result->vrLayer = aLayer;
+
+  return result;
+}
+
+void
+OpenXRLayerPassthrough::Init(JNIEnv *aEnv, XrSession session, vrb::RenderContextPtr &aContext, const XrPassthroughFB& passthroughInstance) {
+  XrPassthroughLayerCreateInfoFB layerCreateInfo = {XR_TYPE_PASSTHROUGH_LAYER_CREATE_INFO_FB};
+  layerCreateInfo.passthrough = passthroughInstance;
+  layerCreateInfo.purpose = XR_PASSTHROUGH_LAYER_PURPOSE_RECONSTRUCTION_FB;
+  layerCreateInfo.flags = XR_PASSTHROUGH_IS_RUNNING_AT_CREATION_BIT_FB;
+  CHECK_XRCMD(OpenXRExtensions::sXrCreatePassthroughLayerFB(session, &layerCreateInfo, &xrLayer));
+}
+
+void
+OpenXRLayerPassthrough::Destroy() {
+  // Release passthrough layer handle
+  if (xrLayer != XR_NULL_HANDLE) {
```

Comment:
Nit: let's convert this in an early return if `xrLayer==XR_NULL_HANDLE`. That way the actual code of the method won't have this extra indentation.